### ISSUE TITLE
Logo Placement

### ISSUE THREAD

**Issue by mjulsen** [yardsalebusiness/Yardsale-#15]
Honestly, the only thing that I think should be changed is the placement of the logo. In my opinion, it should be at the top of the page, either above or below the name. Other than that one small thing, I think everything looks great!

**Comment by lmmuellner** [comment]
I agree that everything looks great but I think that the logo should be placed higher up. So that it is the main view point at the on the page.
